lib/pharos/kubeadm.rb

```diff
@@ -121,23 +125,27 @@ def configure_internal_etcd(config)
           "https://#{h.peer_address}:2379"
         }
         config['etcd'] = {
-          'endpoints' => endpoints
+          'external' => {
+            'endpoints' => endpoints
```

Comment:
Why don't we set `*File` keys here directly?

Reply:
Dunno, kept the existing code structure as-is... I can refactor it at the same time, need to test the external etcd setup anyways.

Reply:
...probably because the external etcd case has all of these as conditionals, and it's a pain to write any other way.